src/webhook/dto/webhook-response.ts

```diff
@@ -0,0 +1,3 @@
+import { Webhook } from '@prisma/client';
+
+export type ListWebhooksDto = Omit<Webhook, 'userId'>;
```

Comment:
hmm, it's kind of risky

advantages:
- you have few lines of code
- you do not have to update this DTO when new fields are added to `Webhook` table

disadvantages:
- you do not have to update this DTO when new fields are added to `Webhook` table - this poses a risk that we leak some data we do not want to. If its used within service then perhaps it's ok.